dev-site-config/navigation.config.js

```diff
@@ -17,6 +17,10 @@ const navConfig = {
       path: '/contributing',
       text: 'Contributing',
       pageTypes: ['contributing'],
+    }, {
+      path: '/issue-form',
+      text: 'Issue Form',
+      pageTypes: ['issueform'],
```

Comment:
Talked with @emilyrohrbough and we can add an `isHidden` key to this config. This will allow us to hide this route from being displayed in the top nav. Then we can [link to it from the footer](https://github.com/cerner/terra-ui/blob/master/src/Footer/Index.jsx#L25).

Reply:
Updated: [d658958](https://github.com/cerner/terra-ui/pull/151/commits/d6589587b241e85489f9f156c29d1dab408f90d0)